servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy

```diff
@@ -170,8 +170,14 @@ final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
         systemProperty "junit.jupiter.extensions.autodetection.enabled", "true"
 
         testLogging {
-          events "passed", "skipped", "failed"
-          showStandardStreams = true
+          events "failed"
+          showStandardStreams = false
```

Comment:
+1, I like the new output on the console, and the generated HTML test reports contain all detailed information.

A couple of suggestions:
1. Use defined enums for `events`.
2. Consider adding `exceptionFormat = FULL` here. It prints a short subset of the stack trace that includes not only the line where the error occurred but also exception type and message:

```
GracefulConnectionClosureHandlingTest > closeIdleAfterExchange[7: protocol=HTTP_2 initiateClosureFromClient=true useUds=false viaProxy=false] FAILED
    java.lang.AssertionError: 
    Expected: is <1>
         but: was <2>
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
        at io.servicetalk.http.netty.GracefulConnectionClosureHandlingTest.closeIdleAfterExchange(GracefulConnectionClosureHandlingTest.java:270)
```

While the `SHORT` (default) mode prints:
```
GracefulConnectionClosureHandlingTest > closeIdleAfterExchange[0: protocol=HTTP_1 initiateClosureFromClient=true useUds=true viaProxy=false] FAILED
    java.lang.AssertionError at GracefulConnectionClosureHandlingTest.java:270
```

Here is my patch:
```patch
Index: servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy b/servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy
--- a/servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy	(revision fc60521dc7ef045364339c9a78cf87f66f9323ce)
+++ b/servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy	(date 1619020228048)
@@ -29,6 +29,11 @@
 import static io.servicetalk.gradle.plugin.internal.Versions.PMD_VERSION
 import static io.servicetalk.gradle.plugin.internal.Versions.SPOTBUGS_VERSION
 import static io.servicetalk.gradle.plugin.internal.Versions.TARGET_VERSION
+import static org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
+import static org.gradle.api.tasks.testing.logging.TestLogEvent.FAILED
+import static org.gradle.api.tasks.testing.logging.TestLogEvent.PASSED
+import static org.gradle.api.tasks.testing.logging.TestLogEvent.SKIPPED
+import static org.gradle.api.tasks.testing.logging.TestLogEvent.STARTED
 
 final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
   void apply(Project project) {
@@ -170,12 +175,13 @@
         systemProperty "junit.jupiter.extensions.autodetection.enabled", "true"
 
         testLogging {
-          events "failed"
+          events = [FAILED]
           showStandardStreams = false
+          exceptionFormat = FULL
 
           warn {
             // Show more complete info when gradle run in --warn mode
-            events "started", "passed", "skipped", "failed"
+            events = [STARTED, PASSED, SKIPPED, FAILED]
             showStandardStreams = true
           }
         }
```